website/docs/reference/strategy-variants.md

```diff
@@ -77,9 +77,9 @@ Strategy stickiness is calculated on the received user and context, as described
 
 If you would like to reassign users to different variants using existing stickiness parameter then you can change the groupId of the strategy. This will provide different input to the stickiness calculation.
 
-### Strategy variants vs feature toggle variants
+### Strategy variants vs feature flag variants
```

Comment:
how do you know that no one references this link?

Reply:
Good catch, to be honest, most of them I just search one by one. But I think this one is actually being referenced.